Classes/TLSLog.h

```diff
@@ -47,7 +47,7 @@ NS_ASSUME_NONNULL_BEGIN
 #pragma mark Convenience Functions
 
 //! Convert the log level to a short parsable string
-FOUNDATION_EXTERN NSString *TLSLogLevelToString(TLSLogLevel level) __attribute__((const));
+FOUNDATION_EXTERN NSString *TLSLogLevelToString(TLSLogLevel level);
```

Comment:
nitpick: why did you remove `__attribute__((const))` from this function, but not from `TLSLogChannelApplicationDefault()` just below ?  don't they both return references to `NSString * const` internally?

Reply:
So, the issue is that `TLSLogLevelToString` was updated to provide a debug string when an invalid level is provided which is dynamically created (not const) and thus if you provide an invalid level you risk crashing from compiler optimization.  It's super unlikely, but it's still a bug and removing the attribute fixes that.